src/Node/Node.php

```diff
@@ -186,17 +186,11 @@ public function removeAttribute(string $name): void
         unset($this->attributes[$name]);
     }
 
-    /**
-     * @param string|int $name
```

Comment:
Int is not accepted

Reply:
It is accepted explicitly in Twig 4.0 (same for all similar methods).